CorsixTH/Src/xmi2mid.cpp

```diff
@@ -31,75 +31,75 @@ SOFTWARE.
 /*!
     Utility class for reading or writing to memory as if it were a file.
 */
-class MemoryBuffer
+class memory_buffer
```

Comment:
looks like something that should be shared and re-used at every point where we have a buffer